src/entities/ActiveITSDBClient.ts

```diff
@@ -0,0 +1,163 @@
+import { ConnectionPool, config } from "mssql";
+export default class ActiveITSDBClient {
+  private static instance: ActiveITSDBClient | null = null;
+  private pool: ConnectionPool | null = null;
+
+  dbConfig: config = {
+    user: "readonlyuser",
+    password: "Caltrans2018",
```

Comment:
Please put credentials in your .env file so passwords and IP addresses aren't in plain text.